hash/fmtowns_cd.xml

```diff
@@ -8596,8 +8638,112 @@ User/save disks that can be created from the game itself are not included.
 		</part>
 	</software>
 
-	<!-- It doesn't seem to be possible to get past the intro -->
-	<software name="kikoshid" supported="no">
+	<software name="parapara">
```

Comment:
The pull request description says this is not working, but it's marked working here.  Which is correct?

Reply:
The correct one is NOT working, I forgot to add the "supported" attribute, sorry for the confusion.

Reply:
Thanks.